app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id] if session[:user_id]
+  end
+
+  def logged_in?
+    current_user.present?
+  end
+
+  def log_out
+    session.delete :user_id
```

Comment:
session hoạt động như thế nào em nhỉ

Reply:
xóa phiên id tài khoản người dùng khi đăng xuất

Reply:
@thanh218 ở đây thì mới là xóa thôi, giải thích giúp anh hoạt động của session theo nghĩa chung nhất nhé

Reply:
Sử dụng session có thể đặt id người dùng một cách an toàn trên trình duyệt để tạo phiên tạm thời (còn theo e hiểu thì session: không được lưu trữ trên trình duyệt người dùng, phía khách chủ, các phiên session sẽ có trên trình duyệt chạy,sau đó đóng chương trình nó sẽ kết thúc phiên mất thông tin)